devtools/Makefile

```
@@ -34,4 +34,11 @@ afl: tools-init
 libfuzzer: tools-init
 	podman run -t --rm -v ${TOPDIR}:/td-shim -w /td-shim tools bash sh_script/fuzzing.sh
 
-all-tools: tools-init afl libfuzzer
+rudra: tools-init
+	mkdir -p ${TOPDIR}/target/report
+	podman run -t --rm -v ${TOPDIR}:/td-shim -v ${TOPDIR}/target/report:/td-shim/target/report  \
+		--env RUDRA_REPORT_PATH=/td-shim/target/report/report \
+		--env SCCACHE_DIR=/td-shim/target/sccache_home --env SCCACHE_CACHE_SIZE=10T \
+		-w /td-shim tools bash sh_script/rudra.sh
```

Nit: formatting improvement:

```suggestion
	podman run -t --rm \
		--env RUDRA_REPORT_PATH=/td-shim/target/report/report \
		--env SCCACHE_DIR=/td-shim/target/sccache_home \
		--env SCCACHE_CACHE_SIZE=10T \
	        -v ${TOPDIR}:/td-shim \
	        -v ${TOPDIR}/target/report:/td-shim/target/report \
		-w /td-shim tools bash sh_script/rudra.sh
```

---

Done